### ISSUE TITLE
host dependency 'glibc-32bit' does not exist!

### ISSUE THREAD

**Issue by boson-at** [void-linux/void-packages#12990]
### System

* xuname:  
 Void 4.19.58_1 x86_64 GenuineIntel uptodate rrmFFF

since about 17 hours I have problems with <depend="... glibc-32bit ...">
( after commit e4984d01ea049e7cde260fc1a05d61fbdd403180 )

srcpkgs/my-test-package/template
```
[ "$XBPS_TARGET_MACHINE" == "x86_64" ] &&  depends+=" glibc-32bit"
```

```
$ ./xbps-src pkg my-test-package
grep: : Datei oder Verzeichnis nicht gefunden
[*] Updating `https://alpha.de.repo.voidlinux.org/current/musl/x86_64-repodata' ...
ERROR: [reposync] failed to fetch file `https://alpha.de.repo.voidlinux.org/current/musl/x86_64-repodata': Not Found
[*] Updating `https://alpha.de.repo.voidlinux.org/current/musl/nonfree/x86_64-repodata' ...
[*] Updating `https://alpha.de.repo.voidlinux.org/current/x86_64-repodata' ...
[*] Updating `https://alpha.de.repo.voidlinux.org/current/nonfree/x86_64-repodata' ...
[*] Updating `https://alpha.de.repo.voidlinux.org/current/aarch64/x86_64-repodata' ...
ERROR: [reposync] failed to fetch file `https://alpha.de.repo.voidlinux.org/current/aarch64/x86_64-repodata': Not Found
[*] Updating `https://alpha.de.repo.voidlinux.org/current/multilib/x86_64-repodata' ...
[*] Updating `https://alpha.de.repo.voidlinux.org/current/multilib/nonfree/x86_64-repodata' ...
=> my-test-package-1.0.1_1: removing autodeps, please wait...
=> my-test-package-1.0.1_1: building ... 
=> ERROR: my-test-package-1.0.1_1: host dependency 'glibc-32bit' does not exist!
```

**Comment by xtraeme** [comment]
I'll take a look at it in a few hours.

**Comment by Johnnynator** [comment]
I edited your post to properly use code tags.
``` ` ``` <- wrap longer logs ore code blocks with 3 backticks (or however they are called :)

**Comment by boson-at** [comment]
@Johnnynator 
Danke, thank you ;)

**Comment by abenson** [comment]
This also seems to break `gcc-multilib`.

**Comment by boson-at** [comment]
the error is probably in "common/xbps-src/shutils/build_dependencies.sh"
I have replaced build_dependencies.sh with the version before "commit e4984d01ea049e7cde260fc1a05d61fbdd403180" (529a019a3f29df09936cbb12179c2eb31bcc2e27) 

and "./xbps-src pkg my-test-package" works propperty

**Comment by MrPhantom** [comment]
Any updates on this? gcc-multilib is a version behind gcc and building is still broken.

**Comment by jnbr** [comment]
fixed in #14822
